src/fossa_cli.rs

```diff
@@ -123,11 +209,15 @@ async fn download(config_dir: &PathBuf) -> Result<PathBuf, Error> {
             .wrap_err()
             .context(Error::FindVersion);
     }
-    let version = tag.trim_start_matches('v');
+    Ok(tag.trim_start_matches('v').to_string())
```

Comment:
[optional] this reads slightly more cleanly IMO:
```suggestion
    tag.trim_start_matches('v').to_string().wrap_ok()
```